docs/standard/security-analyzer.md

```diff
@@ -0,0 +1,62 @@
+---
+title: The .NET Security Analyzers - .NET
+description: Learn how to use the .NET Security Analyzers in the .NET Desktop Analyzers package to find and address security risks
+keywords: .NET, .NET Core
+author: billwagner
+ms.author: billwagner
+ms.date: 01/10/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# The .NET Security Analyzers
+
+You can use the .NET Security Analyzers to find potential security risks
+in your code. These analyzers will find potential issues and suggest
+fixes to those issues.
+
+The analyzers run interactively in Visual Studio as you write your code
+or as part of a CI build. You'll want to add the analyzer to your project as
+early as possible in your development. The sooner you find any potential issues
+in your code, the easier it will be to fix. However, you can add it at any time
+in the development cycle. It will find any issues with the existing code and
+will start warning about new issues as you keep developing.
+
+## Installing and configuring the .NET Security Analyzers
+
+The .NET Security Analyzers must be installed as a NuGet package on every
+project where you want them to run. Only one developer needs to add them
+to the project. The analyzer package is a project dependency and will run
+on every developer's machine once they have the updated solution.
+
+The .NET Security Analyzers are delivered in the [Microsoft.DesktopAnalyzers](https://www.nuget.org/packages/Desktop.Analyzers/)
```

Comment:
Desktop analyzers package is no longer maintained, and we should not recommend anyone to install them. We have refactored the analyzers into following packages:

**Core analyzer packages**
1. [Microsoft.CodeQuality.Analyzers](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers): Common code quality rules, which are not specific to any .net framework or APIs. See documentation for rules included in this package [here](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeQuality.Analyzers/Microsoft.CodeQuality.Analyzers.md)
2. [Microsoft.NetCore.Analyzers](https://www.nuget.org/packages/Microsoft.NetCore.Analyzers): Analyzers specific to netcore APIs. See documentation for rules included in this package [here](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.NetCore.Analyzers/Microsoft.NetCore.Analyzers.md)
3. [Microsoft.NetFramework.Analyzers](https://www.nuget.org/packages/Microsoft.NetFramework.Analyzers): Analyzers specific to netfx APIs. See documentation for rules included in this package [here](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.NetFramework.Analyzers/Microsoft.NetFramework.Analyzers.md)
4. [Text.Analyzers](https://www.nuget.org/packages/Text.Analyzers): Analyzers specific to text APIs, e.g. for analyzer and code fix authors. See documentation for rules included in this package [here](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Text.Analyzers/Text.Analyzers.md)
5. [Microsoft.CodeAnalysis.Analyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers): Analyzers specific to usage of Roslyn/Microsoft.CodeAnalysis APIs, e.g. for analyzer and code fix authors. See documentation for rules included in this package [here](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md)

**Aggregate analyzer packages**
1. [Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/): Analyzers aimed at replacement of legacy FXCop rules. This package is an aggregate of the following analyzer packages:
   1. Microsoft.CodeQuality.Analyzers
   2. Microsoft.NetCore.Analyzers
   3. Microsoft.NetFramework.Analyzers
   4. Text.Analyzers

Reply:
I'll expand this PR to cover NetFramework.Analyzers.  I've opened the following new issues for the others that are part of the FxCopAnalyzers package:

#4214
#4215
#4216

Note that the CodeAnalysis package will be covered as part of the Roslyn API documentation.